nativescript-angular/directives/dialogs.ts

```diff
@@ -81,7 +82,10 @@ export class ModalDialogService {
         const componentContainer = moduleRef || viewContainerRef;
         const resolver = componentContainer.injector.get(ComponentFactoryResolver);
 
-        const frame = parentView.page && parentView.page.frame;
+        let frame = parentView;
```

Comment:
Can we add a comment about the scenario where parentView is a Frame? (it seems previously there wasn't such scenario?)